src/test/java/com/autonomouslogic/commons/rxjava3/Rx3UtilTest.java

```diff
@@ -58,14 +113,46 @@ void shouldConvertCompletionStageNullResultToMaybe() {
 		assertEquals("empty", result);
 	}
 
+	@Test
+	void confirmCatchMaybeFromFuture() {
+		// This test confirms how RxJava's fromFuture works, so it can be replicated below
+		var future = CompletableFuture.failedFuture(textEx);
+		var maybe = Maybe.fromFuture(future);
+		var ex = assertThrows(RuntimeException.class, maybe::blockingGet);
+		ex.printStackTrace();
+		assertEquals("test error", ex.getMessage());
```

Comment:
I don't know why RxJava's `Maybe.fromFuture` doesn't wrap it like the others do. I opted to wrap it below to have consistency.